src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -203,33 +205,28 @@ public static String getIndividualFullName(Entity entity) {
         return entity.getName().getFullNameNoTitle();
     }
 
-    public static String getIndividualPreferredHearingChannel(String hearingType,
-                                                                        HearingSubtype hearingSubtype,
+    public static HearingChannel getIndividualPreferredHearingChannel(HearingSubtype hearingSubtype,
                                                                         HearingOptions hearingOptions) {
-        if (eitherNull(hearingType, hearingSubtype)) {
+        if (isNull(hearingSubtype) || isNull(hearingOptions)) {
             return null;
         }
 
         HearingChannel preferredHearingChannel =
-            shouldPreferNotAttendingHearingChannel(hearingType, hearingOptions) ? NOT_ATTENDING
+            shouldPreferNotAttendingHearingChannel(hearingOptions) ? NOT_ATTENDING
             : isYes(hearingSubtype.getWantsHearingTypeFaceToFace()) ? FACE_TO_FACE
             : shouldPreferVideoHearingChannel(hearingSubtype) ? VIDEO
             : shouldPreferTelephoneHearingChannel(hearingSubtype) ? TELEPHONE
             : null;
 
-        if (preferredHearingChannel == null) {
+        if (isNull(preferredHearingChannel)) {
             throw new IllegalStateException("Failed to determine a preferred hearing channel");
         }
 
-        return preferredHearingChannel.getHmcReference();
-    }
-
-    private static boolean eitherNull(String hearingType, HearingSubtype hearingSubtype) {
-        return hearingType == null || hearingSubtype == null;
+        return preferredHearingChannel;
     }
 
-    private static boolean shouldPreferNotAttendingHearingChannel(String hearingType, HearingOptions hearingOptions) {
-        return PAPER.getHmcReference().equals(hearingType) || !hearingOptions.isWantsToAttendHearing();
+    private static boolean shouldPreferNotAttendingHearingChannel(HearingOptions hearingOptions) {
+        return !hearingOptions.isWantsToAttendHearing();
     }
```

Comment:
Your already calling a method isWantsToAttendHearing(), I don't think you need another one to call that method

Reply:
Good shout I just used the method there but is removed now